File: CS50x_computer_science/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
int h;
do
{
     h = get_int ("HEIGHT?: ");
}
while (h<1 || h>8);

for(int i = 0; i<h; i++)
{
                for(int d=8; d>i; d--){
                        printf(" ");
                }
       for(int j=0; j<i;j++)
{
        printf("#");
}
           printf("\n");
}
}

File: CS50x_computer_science/sentimental-mario-more/mario.py
from cs50 import get_int

def main():
    while True:
        n = get_int("Height: ")
        if (n < 1 and n > 8):
            break
        print_dash(n)

def print_dash(n):
    while True:
        print("#")
        n = n -1

        if (n == 0):
            break

main()